src/net.h

```diff
@@ -928,9 +922,6 @@ class CNode
     // m_tx_relay == nullptr if we're not relaying transactions with this peer
     std::unique_ptr<TxRelay> m_tx_relay;
 
-    // Used for headers announcements - unfiltered blocks to relay
```

Comment:
What's the distinction you loose by dropping `unfiltered blocks to relay` ?

I guess this is a reference to sort done at headers announcement, L4218 src/net_processing.cpp:
https://github.com/bitcoin/bitcoin/blob/1cf73fb8eba3b89a30d5e943deaec5052a8b21b7/src/net_processing.cpp#L4218

Before to announce headers, we check both with regards to peer's context ("Does this header connect to known peer's tip ?") and our context ("Does this header is part of best chain ?"). It wasn't a clear comment, so yes either drop it or extend ?

Reply:
I think drop it.